packages/editor/src/store/effects/reusable-blocks.js

```diff
@@ -237,7 +237,11 @@ export const convertBlockToStatic = ( action, store ) => {
 	if ( referencedBlock.name === 'core/template' ) {
 		newBlocks = referencedBlock.innerBlocks.map( ( innerBlock ) => cloneBlock( innerBlock ) );
 	} else {
-		newBlocks = [ createBlock( referencedBlock.name, referencedBlock.attributes ) ];
+		newBlocks = [ createBlock(
+			referencedBlock.name,
+			referencedBlock.attributes,
+			referencedBlock.innerBlocks
```

Comment:
I think we should clone the blocks before calling this:

 - Insert the reusable block twice
 - Convert one of them to regular
 - Edit the regular blocks (notice they change in both blocks)